src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -47,70 +47,82 @@ NRQL requirements:
 
 To set up streaming data export to AWS, you must first set up an Amazon Kinesis Firehose. We'll walk you through that procedure in the next three steps.
 
-### Step 1. Create a Firehose for streaming export [#create-firehose]
+<Steps>
+  <Step>
+### Create a Firehose for streaming export [#create-firehose]
 
 Create a dedicated Firehose to stream your New Relic data to:
 
 1. Go to Amazon Kineses Data Firehose.
 2. Create a delivery stream.
 3. Name the stream (you will use this name later in the rule registration).
 4. Use <DNT>**Direct PUT or other sources**</DNT> and specify a destination compatible with New Relic's JSON event format (for example, S3, Redshift, or OpenSearch).
-
-### Step 2. Create IAM Firehose write access policy [#create-policy]
+  </Step>
+  <Step>
+### Create IAM Firehose write access policy [#create-policy]
 
 1. Go to the IAM and select <DNT>**Policies**</DNT>.
```

Comment:
I'd modify these steps:
1. Go to the IAM console and sign in with your user.
2. Click **Policies** in the left navigation and then click **Create policy**.

and step 6 is the same as step 2?

Reply:
@nbaenam I used your two suggestions for the first two steps.

I think Step 6 should probably be "Save the policy" but I don't have access to this. I think readers will understand.